lib/MetaCPAN/Web/Controller/Pod.pm

```diff
@@ -40,6 +40,7 @@ sub view : Private {
     my $release    = $c->stash->{release};
 
     if ( $data->{directory} ) {
+        $c->cdn_max_age('1y');
```

Comment:
This can end up being a redirect from a non-versioned path to a versioned path, and it doesn't have any surrogate keys so it won't get purged.